notebooks/feature_engineering/raw/ex1.ipynb

```diff
@@ -278,7 +278,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This will be our baseline score for the model. When we transform features, add new ones, or perform feature selection, we should be improving on this score. However, since this is the test set, we only want to look at it at the end of all our manipulations. At the very end of this course you'll look at the test score again to see if you improved on the baseline model."
+    "This will be our baseline score for the model. When we transform features, add new ones, or perform feature selection, we should be improving on this score. However, since this is the test set, we only want to look at it at the end of all our manipulations. At the very end of this course you'll look at the test score again to see if you improved on the baseline model.\n",
+    "\n",
+    "# Keep Going\n",
+    "Now that you have a baseline model, you are ready to learn **[Categorical Encoding Techniques](#$NEXT_NOTEBOOK_URL$)** to improve it."
```

Comment:
Need link to next lesson. done with NEXT_NOTEBOOK_URL macro

Reply:
Great, thanks!